spec/requests/graphql/mutations/term/create_term_spec.rb

```diff
@@ -0,0 +1,253 @@
+            termType: 'external',
+            customFields: [
+              { field: "classification", value: "Human" }
+            ]
+          }
+        }
+      end
+      let(:expected_response) do
+        %({
+            "term": {
+              "prefLabel": "Minotaur (Greek mythological character)",
+              "altLabels": [],
+              "uri": "http://id.worldcat.org/fast/1023481",
+              "authority": "fast",
+              "termType": "external",
+              "customFields": [
+                { "field": "classification", "value": "Human" },
+                { "field": "harry_potter_reference", "value": null}
+              ]
+            }
+          })
+      end
+
+      before { graphql query, variables }
+
+      it 'creates a term record' do
+        expect(Term.count).to be 1
+        expect(Term.first.uri).to eql 'http://id.worldcat.org/fast/1023481'
+      end
+
+      it 'returns newly created term in json' do
+        expect(response.body).to be_json_eql(expected_response).at_path('data/createTerm')
+      end
+    end
+
+    context 'when successfully creating a new local term' do
+      let(:variables) do
+        {
+          input: {
+            vocabularyStringKey: vocabulary.string_key,
+            prefLabel: 'Hippogriff',
+            altLabels: ['Hippogryph'],
+            termType: 'local',
+            customFields: [
+              { field: 'classification', value: 'Eagle' }
+            ]
+          }
+        }
+      end
+      let(:expected_response) do
+        %({
+            "term": {
+              "prefLabel": "Hippogriff",
+              "altLabels": ["Hippogryph"],
+              "termType": "local",
+              "authority": null,
+              "customFields": [
+                { "field": "classification", "value": "Eagle" },
+                { "field": "harry_potter_reference", "value": null }
+              ]
+            }
+          })
+      end
+
+      before { graphql query, variables }
+
+      it 'creates a term record' do
+        expect(Term.count).to be 1
+        expect(Term.first.uri).not_to be_blank
+        expect(Term.first.alt_labels.first).to eql 'Hippogryph'
+      end
+
+      it 'returns newly created term in json' do
+        expect(response.body).to be_json_eql(expected_response).at_path('data/createTerm').excluding('uri')
+      end
+    end
+
+    context 'when successfully creating a new temporary term' do
+      let(:variables) do
+        {
+          input: {
+            vocabularyStringKey: vocabulary.string_key,
+            prefLabel: 'Hippogriff',
+            termType: 'temporary'
+          }
+        }
+      end
+
+      let(:expected_json) do
+        %({
+            "term": {
+              "prefLabel": "Hippogriff",
+              "altLabels": [],
+              "authority": null,
+              "termType": "temporary",
+              "customFields": [
+                { "field": "classification", "value": null },
```

Comment:
It might be a good idea to ensure/test that customFields always come back the same order, for consistency.  Alphabetical order, based on field name.  If you agree, we can ticket that as future work.

Reply:
They are already returned in alphabetical order based on field name. TermType sorts them.

Reply:
Oh good!  Never mind then!  Is it worth writing a test to ensure they're sorted, or no?  I'm fine with skipping if you don't think so.

Reply:
I think it's okay to skip, the tests are going to start failing if we don't sort anyways.